tests/unittests/tests-nanocoap/tests-nanocoap.c

```diff
@@ -250,6 +254,108 @@ static void test_nanocoap__get_path_too_long(void)
     TEST_ASSERT_EQUAL_INT(-ENOSPC, get_len);
 }
 
+/*
+ * Helper for server_get tests below.
+ * Request for nanocoap server example /riot/value resource.
+ * Includes 2-byte token; non-confirmable.
+ */
+static int _read_riot_value_req(coap_pkt_t *pkt, uint8_t *buf)
+{
+    uint8_t pkt_data[] = {
```

Comment:
where do these come from, I mean what's the content,  how is it generated?

Reply:
mhm, okay the comment says something, but still how is it "generated" -> captured?

Reply:
I probably used libcoap to generate it, but I'm sure I reviewed it in Wireshark. I'll add a note to that effect.